tests/python/pants_test/backend/python/tasks/native/test_ctypes_integration.py

```diff
@@ -26,7 +27,7 @@ def invoke_pex_for_output(pex_file_to_run):
 def _toolchain_variants(func):
   @wraps(func)
   def wrapper(*args, **kwargs):
-    for variant in ToolchainVariant.all_values():
+    for variant in ToolchainVariant.__members__.values():
```

Comment:
I think you can simply say `for variant in ToolchainVariant`. I’m on my phone and not 100% sure this will work, but worth trying.

Reply:
+1 you can iterate an Enum type to get all its members.